Sitka-Sound-Science-Center/sitka-landslide
# Copy changes (plus some formatting)

## Overview

This PR makes the following copy changes as requested by the client:
- Adds "Pink" to the area definitions of "About the legend" section of the "Areas at risk" page
- Changes the NWS alert banner to say "Weather alert from NWS"
- Changes the title of the Oral histories page and adds a paragraph to the bottom
- Adds a bullet to the "how to prepare" section of the "how can I reduce my risk" list. This also required an addition of an `em` selector to the `reset.css` file, since prior to this italics would be overridden by that file
- Removes the word "beta" from the banner

It also adds a note to the `README` regarding silent failures when trying to run scripts for Mac users and applies formatting to a few files that Prettier deemed necessary.

Resolves #62 

## Testing Instructions

* Start the server
* Go through the list in [this document](https://docs.google.com/document/d/1u70SHNwtt_eW8bPwOaeD8zywl1QTKk3zsGdCvgt1xXw/edit#heading=h.dqrdzjtok98y) and make sure that all the copy changes requested have been applied


## Checklist
